### PR TITLE
Kitchen tweaks

### PR DESCRIPTION
**Pull request by Lohikar**
changes:
- Reduced power usage of oven and fryer.
- Reduced warm up time of oven and fryer.
- Oven now heats up to exactly 200C instead of to 198.85C.
- Fixed some issues with cooking appliances' sanity checks.
- Recipe datums are now initialized once instead of per-appliance.
- Moved some cooking verbs to the Objects tab instead of a null category.

### CODE REVIEW

File: code/game/machinery/kitchen/cooking_machines/oven.dm [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
 	food_color = "#A34719"
 	can_burn_food = 1
-	active_power_usage	= 19000
+	active_power_usage = 4 KILOWATTS
 	//Based on a double deck electric convection oven
 
-	resistance = 72000
-	idle_power_usage	= 6000
+	resistance = 10000
+	idle_power_usage = 6000
```
Comment (Arrow768) [inline code/game/machinery/kitchen/cooking_machines/oven.dm]: Takes 4KW to heat up and 6KW to keep the temperature ?
Dont just go to 20% of the initial value when changing stats of a item.
Use smaller steps and repeat until finding the optimal value.

File: code/game/machinery/kitchen/cooking_machines/fryer.dm [hunk 2]
```diff
@@ -9,17 +9,15 @@
 	food_color = "#FFAD33"
 	cooked_sound = 'sound/machines/ding.ogg'
 	appliancetype = FRYER
-	active_power_usage	= 24000
-	//24 KW, based on real world values for a large commercial fryer.
+	active_power_usage = 8 KILOWATTS
```
Comment (Arrow768) [inline code/game/machinery/kitchen/cooking_machines/fryer.dm]: Use smaller increments when tweaking values.

**Comment by Lohikar** [comment]
I mean, it's not a particularly big oven, no? Even after this nerf, the kitchen can still use over 20 kW.

File: code/game/machinery/kitchen/cooking_machines/oven.dm (hunk 1 above)
Comment (Lohikar) [inline code/game/machinery/kitchen/cooking_machines/oven.dm]: Oops, forgot to update the idle usage.

File: code/game/machinery/kitchen/cooking_machines/oven.dm [hunk 3]
```diff
@@ -46,33 +46,28 @@
...
-	if(user.stat || user.restrained())	return
-	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)//No spamming the door, it makes a sound
-	toggle_door()
-
+	try_toggle_door(user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
 
 /obj/machinery/appliance/cooker/oven/verb/toggle_door()
-	set src in view()
+	set src in orange(1)
```
Comment (skull132) [inline code/game/machinery/kitchen/cooking_machines/oven.dm]: `orange` is rarely the correct use, `oview` please. Range doesn't consider visibility.

**Pull request merged**